guides/source/form_helpers.md

```diff
@@ -999,7 +999,7 @@ remove addresses:
 <% end %>
 ```
 
-Don't forget to update the whitelisted params in your controller to also include
+Don't forget to update the permitted params in your controller to also include
```

Comment:
By renaming whiltelist to:
* permitted
* allowed
* trusted

you're only creating more confusion.

Reply:
I think we could reword this to consistently use "permit" or "permitted".

Reply:
@tenderlove Roger that.